src/main/java/nextstep/subway/path/ui/PathController.java

```diff
@@ -18,8 +20,10 @@ public PathController(final PathService pathService) {
     }
 
     @GetMapping("/paths")
-    public ResponseEntity<PathResponse> findShortestPath(@RequestParam final Long sourceId, @RequestParam final Long targetId) {
-        return ResponseEntity.ok(pathService.findShortestPath(sourceId, targetId));
+    public ResponseEntity<PathResponse> findShortestPath(@AuthenticationPrincipal LoginMember loginMember,
```

Comment:
```suggestion
    public ResponseEntity<PathResponse> findShortestPath(@AuthenticationPrincipal(required = false) LoginMember loginMember,
```

로그인 유저가 없을 경우, `LoginMember.Guest()` 같은 객체를 반환해서 계산하도록 해도 되겠네요 :thinking: 


![image](https://user-images.githubusercontent.com/12059592/205479747-1aab68a4-74a6-4a87-9b0f-b923f07f76c7.png)

Reply:
음 이부분을 제가 놓쳤네요 😢 
수정하도록 하겠습니다 ~!